UI/src/app/modules/courses-page/components/course-description/course-description.component.ts

```diff
@@ -45,40 +44,38 @@ export class CoursesDescriptionComponent implements OnInit {
   }
 
   public submit(): void {
-    const course: ICourse = this.courseDescription.value;
-    course.creationDate = this.parseDateString(course.date);
-    course.duration = parseInt(course.duration.toString(), 10);
+    const courseValue = this.courseDescription.value;
+    const id = this.courseId ? this.courseId : null;
```

Comment:
const id = this.courseId ? this.courseId : null; => const id = this.courseId || null;